### PR TITLE
java version as variable

### PR DESCRIPTION
**Pull request by svrnm**
We have the same for the collector getting started, makes it easier to update (and it's one step closer towards automation)

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]
That's awesome! 🚀 
With this in place we'll be able to implement post-release automation pretty easily.

Also, there's another place where the project version is mentioned: it's the `/java/automatic/annotations/` page

**Comment by svrnm** [comment]
> That's awesome! 🚀 With this in place we'll be able to implement post-release automation pretty easily.

If you know how to do that, that would be awesome, we could make use of it for the other SIGs where applicable as well.

> Also, there's another place where the project version is mentioned: it's the `/java/automatic/annotations/` page

added, please have a look

**Comment by mateuszrzeszutek** [comment]
> If you know how to do that, that would be awesome, we could make use of it for the other SIGs where applicable as well.

Yeah, it would look very similar to e.g. https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/reusable-create-java-instrumentation-pull-request.yml

File: content/en/docs/instrumentation/java/automatic/annotations.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Annotations
 description: Using instrumentation annotations with a Java agent.
 aliases: [/docs/instrumentation/java/annotations]
 weight: 4
+javaAnnotationsVersion: 1.17.0-alpha
```
Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/automatic/annotations.md:6]: We might as well bump it right now; the new instrumentation version was released yesterday 😄 
```suggestion
javaAnnotationsVersion: 1.18.0-alpha
```

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Comment by trask** [comment]
I sent automation:

* https://github.com/open-telemetry/opentelemetry-java/pull/4765, and
* https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6650

will test them out in a few weeks with the 1.19.0 release

**Comment by cartermp** [comment]
Oh, really nice @trask!

**Comment by trask** [comment]
teamwork 👏

**Pull request merged**